src/sass/colors/_colors-helpers.scss

```diff
@@ -26,8 +26,8 @@ Styleguide Colors.Helpers
     color: $colorValue;
   }
   .fill-#{$color} {
-    fill: $colorValue;
-    stroke: $colorValue;
+    fill: $colorValue !important;
```

Comment:
There's no need to set these as `!important`, it's enough that they are set and the user of the icons should be able to overwrite them if needed.